src/clj/rems/api.clj

```diff
@@ -72,6 +74,12 @@
   (-> (service-unavailable "please try again")
       (plain-text)))
 
+(defn data-exception-handler
+  [exception _ex-data _request]
+  (log/error "data exception" (pr-str (.-data exception)))
+  (-> (service-unavailable (json/generate-string (.-data exception)))
```

Comment:
nice

Reply:
how about one api-level test that exercises this?

Reply:
yeah I can make one, also perhaps later we'll have other DataExceptions